cli/tests/default/unit/test_rule.py

```diff
@@ -19,9 +24,11 @@ def create_validator_rule(validation_state_action: str) -> Rule:
           message: bad
           metadata:
             dev.semgrep.validation_state.actions:
-              valid: {validation_state_action}
-              invalid: comment
-              error: disabled
+              valid: {valid_action}
+              invalid: {invalid_action}
+              error: {error_action}
+            dev.semgrep.actions:
+              - {action}
```

Comment:
Is this a list?

Reply:
Seems like it is from elsewhere but unclear why to me.

Reply:
Yup this is a list. I don't have the full context but `dev.semgrep.actions` has been a list despite only holding one action.

Reply:
Happy to chat offline about it.